wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/MessageActions/MessageActionsViewController.swift

```diff
@@ -26,7 +26,8 @@ class MessageActionsViewController: UIAlertController {
 
     static func controller(withActions actions: [MessageAction],
                            actionController: ConversationMessageActionController) -> MessageActionsViewController {
-        let controller = MessageActionsViewController(title: MessageLabelMarker,
+        let title = actionController.canPerformAction(action: .react(.like)) ? MessageLabelMarker : nil
+        let controller = MessageActionsViewController(title: title,
```

Comment:
Why do you want to create a `MessageActionsViewController` with `nil` title and message if it's `.react(.like)`? Why not make this function return an optional result?

Reply:
Title label is a placeholder for Reaction Picker.  If we can't add reaction then we don't need this title label. So I set nil for a title.

Even if we don't show reaction picker we're adding UIAlertActions to that alert controller. That's why this function doesn't returning nil